src/momento-redis-adapter.ts

```diff
@@ -197,7 +202,7 @@ export class MomentoRedisAdapter
     } else if (rsp instanceof CacheGet.Error) {
       this.emitError('get', rsp.message(), rsp.errorCode());
     } else {
-      this.emitError('get', 'unexpected-response');
+      this.emitError('get', 'unexpected-response ' + typeof rsp);
```

Comment:
not a blocker for this PR but did you run this locally and observe this showing useful info? I'd expect it to just say `Object`. I think you might have better luck calling `rsp.toString()`?